critiquebrainz/frontend/external/spotify.py

```diff
@@ -6,12 +6,55 @@
 import requests
 import urllib.parse
 from brainzutils import cache
+from base64 import b64encode
+from flask import current_app as app
 
 DEFAULT_CACHE_EXPIRATION = 12 * 60 * 60  # seconds (12 hours)
+ACCESS_TOKEN_EXPIRATION = 60 * 60 # seconds (1 hour)
 
 BASE_URL = "https://api.spotify.com/v1"
 
 
+def fetch_access_token():
```

Comment:
Make sure to prefix functions that are not meant to be used outside of the module with `_`.